File: src/test/java/com/github/qky666/selenidepom/test/java/mtp/BaseMtpDesktopTest.java
package com.github.qky666.selenidepom.test.java.mtp;

import com.github.qky666.selenidepom.Config;
import org.junit.jupiter.api.BeforeAll;

public abstract class BaseMtpDesktopTest extends BaseMtpTest {
    @BeforeAll
    static void setUpDesktop() {
        System.clearProperty("chromeoptions.mobileEmulation");
        Config.INSTANCE.setPomVersion("desktop");
    }
}

File: src/test/kotlin/com/github/qky666/selenidepom/test/kotlin/mtp/BaseMtpDesktopTest.kt
package com.github.qky666.selenidepom.test.kotlin.mtp

import com.github.qky666.selenidepom.Config
import org.junit.jupiter.api.BeforeAll

abstract class BaseMtpDesktopTest : BaseMtpTest() {
    companion object {
        @BeforeAll
        @JvmStatic
        fun setUpDesktop() {
            System.clearProperty("chromeoptions.mobileEmulation")
            Config.pomVersion = "desktop"
        }
    }
}
